FlowCryptUI/Nodes/AttachmentNode.swift

```diff
@@ -36,6 +42,20 @@ public final class AttachmentNode: CellNode {
         buttonNode.setImage(UIImage(named: "download")?.tinted(.gray), for: .normal)
         titleNode.attributedText = input.name
         subtitleNode.attributedText = input.size
+        
+        buttonNode.addTarget(self, action: #selector(onDownloadButtonTap), forControlEvents: .touchUpInside)
+        
+    }
+    
+    public override func layoutDidFinish() {
```

Comment:
@Kharchevskyi here is the fix that I was talking about

Reply:
Please check next commit with better approach for this. Please confirm this one is works for you